plenum/test/checkpoints/test_stashed_messages_processed_on_backup_replica_ordering_resumption.py

```diff
@@ -0,0 +1,119 @@
+from plenum.server.replica import Replica
+from plenum.test import waits
+from plenum.test.delayers import cDelay, chk_delay
+from plenum.test.helper import sdk_send_random_requests, assertExp
+from stp_core.loop.eventually import eventually
+
+nodeCount = 4
+
+CHK_FREQ = 5
+
+# LOG_SIZE in checkpoints corresponds to the catch-up lag in checkpoints
+LOG_SIZE = 2 * CHK_FREQ
+
+
+def test_stashed_messages_processed_on_backup_replica_ordering_resumption(
```

Comment:
Hos this test is different comparing to test_backup_replica_resumes_ordering_on_lag_if_checkpoints_belate besides different LOG_SIZE  value?

Reply:
The scenario is the same but 3PC-messages from the last checkpoint get into the watermarks-based stash due to the different `LOG_SIZE`.